libs/icons/src/components/sage-icon/sage-icon.scss

```diff
@@ -23,11 +28,11 @@ svg {
   width: 100%;
 }
 
-/* Icon Sizes
- * -----------------------------------------------------------
- */
+// Icon Sizes
+// -----------------------------------------------------------
+//
 
- :host(.sage-icon-small) {
+:host(.sage-icon-small) {
```

Comment:
The icon size looks like it's being determined by the inline `style`s, does this section still need to exist?